.github/scripts/setup-workflow-monitor.py

```diff
@@ -30,26 +24,44 @@ def setup_workflow_monitor(platform: Platform, max_runtime: int) -> None:
         # Azure jobs. On testing, the environment variables did not correctly pass themselves to the
         # screen job on
 
+        env_dict = ci_env.copy()
         if platform == Platform.AZURE:
```

Comment:
since you've removed the env.sh buffoonery I did, you don't need the azure credited env anymore.